portals/devportal/source/src/app/components/Apis/Details/GraphQLConsole/AddEditAdditionalHeaders.jsx

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import { useIntl, FormattedMessage } from 'react-intl';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import FormDialogBase from 'AppComponents/Apis/Details/GraphQLConsole/FormDialogBase';
+import TextField from '@material-ui/core/TextField';
+import CreateIcon from '@material-ui/icons/Create';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme) => ({
+    error: {
+        color: theme.palette.error.dark,
+    },
+}));
+
+/**
+ * Render delete dialog box.
+ * @param {JSON} props component props.
+ * @returns {JSX} Loading animation.
+ */
+function AddEditAdditionalHeaders(props) {
+    const intl = useIntl();
```

Comment:
Shall we use the `FormattedMessage` component instead of `useIntl`

```js
import { FormattedMessage } from 'react-intl';
```

Reply:
Fixed